### ISSUE TITLE
Implement Coverity checker and include triaged defects

### ISSUE THREAD

**Issue by JasperCraeghs** [melexis/warnings-plugin#128]
We want to complete https://github.com/melexis/warnings-plugin/pull/74 and add support for this checker to the [code-quality-report](https://github.com/melexis/warnings-plugin#code-quality-report) feature.

The classification of the defect shall determine the [`severity` string](https://docs.gitlab.com/ee/ci/testing/code_quality.html#implement-a-custom-tool). Every classification included in this configuration, will be included in the code quality report, and, by default, they will be counted towards the warnings limit. You can exclude a classification from the count. Any classification not included in this configuration, will be ignored. Example configuration:

```
{
    "intentional": {"severity": "info", "count": False},
    "bug": {"severity": "major"},
    "pending": {"severity": "critical"},
}
```
What do you think, @Letme ?

**Comment by Letme** [comment]
Sounds what we want yes and also severity is OK.

**Comment by JasperCraeghs** [comment]
The syntax of the configuration was implemented differently. This issue has been resolved in #147
